libs/core/src/lib/product-switch/product-switch-body/product-switch-body.component.scss

```diff
@@ -40,6 +40,13 @@ $product-switch-dnd-line-color: #0854A0;
     &.fd-dnd-on-drag {
         background-color: white;
         z-index: 15;
+        .fd-product-switch__item {
+            .fd-product-switch__title {
+                &:before, &:after {
+                    background-color: white;
```

Comment:
there should be a theming parameter for that

Reply:
The whole file has hard-coded values

Reply:
I see - we should address it in a separate issue